app/src/androidTest/java/com/github/scribeWizTeam/scribewiz/HomeFragmentTest.kt

```diff
@@ -0,0 +1,29 @@
+package com.github.scribeWizTeam.scribewiz
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.fragment.app.testing.launchFragmentInContainer
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.scribeWizTeam.scribewiz.Activities.MainActivity
+import com.github.scribeWizTeam.scribewiz.Fragments.HomeFragment
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class HomeFragmentTest {
+
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<MainActivity>()
+
+    @Test
+    fun homeFragment_DisplayHomeTitle() {
```

Comment:
Do we not want to launch all fragments in a before function in tests like this to avoid redondent code and have a more unified test set ?